src/main/java/com/cinema/shop/Main.java

```diff
@@ -59,12 +61,12 @@ public static void main(String[] args) throws AuthenticationException {
 
         List<MovieSession> availableSessions =
                 movieSessionService.findAvailableSessions(movie2.getId(), LocalDate.now());
-        availableSessions.forEach(System.out::println);
+        availableSessions.forEach(log::info);
 
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authenticationService.register("chester@mail.com", "1234f");
-        System.out.println(authenticationService.login("chester@mail.com", "1234f"));
+        log.info(authenticationService.login("chester@mail.com", "1234f"));
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block